commit 92ec0f5cfb904833ab04a439b09e2206386fd872 (refs/changes/91/2191/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-07-27T23:51:17+00:00 (2 years, 2 months ago)
    
    MANATEE-335 Ship both Postgres 9.2 and 9.6 in manta-manatee images
    Reviewed by: David Pacheco <dap@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/targets.json.in b/targets.json.in
index 6df3b2f..8d8da58 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -943,15 +943,10 @@ cat <<EOF
     "build_platform": "20141030T081701Z",
     "image_name": "manta-postgres",
     "image_description": "Manta manatee",
-    "image_uuid": "b4bdc598-8939-11e3-bea4-8341f6861379",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-        "postgresql92-server-9.2.4nb1",
-        "postgresql92-adminpack",
-        "postgresql92-datatypes",
-        "postgresql92-pgcrypto",
-        "postgresql92-replicationtools",
-        "postgresql92-upgrade",
-        "lz4-120"
+        "lz4-131nb1"
     ],
     "repos": [
       {"url": "git@github.com:joyent/manta-manatee.git"}
diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index ccddabd..b661929 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -317,7 +317,7 @@ if [[ -n "${packages}" ]]; then
   ${SSH} "/opt/local/bin/pkgin -f -y update"
   ${SSH} "touch /opt/local/.dlj_license_accepted"
 
-  if [[ ${build_name} == "manatee" || ${build_name} == "sdc-manatee" || ${build_name} == "manta-manatee" ]]; then
+  if [[ ${build_name} == "manatee" || ${build_name} == "sdc-manatee" ]]; then
     ${SSH} "/opt/local/bin/pkgin -y remove libuuid"
   fi
 
